ui/guest/src/react/GuestProxy.tsx

```diff
@@ -196,23 +199,22 @@ export function GuestProxy() {
         document.querySelectorAll<HTMLElement>('[data-craftercms-model-id]').forEach(registerElement);
       });
 
-    const handler: JQuery.EventHandlerBase<any, any> = (e: Event): void => {
-      let record = ElementRegistry.fromElement(e.currentTarget as Element);
+    const handler: JQuery.EventHandlerBase<any, any> = (e: Event, target): void => {
```

Comment:
Fix types. This should no longer be a JQuery.Event.
This type should be giving some sort of typescript error. Otherwise, there may still be some jquery typings or something on the env.

Make sure you're running yarn build:modern to detect typescript errors

Reply:
the reason why there's a @types/jquery is because it is a dep of @types/tinymce@npm. Types fixed